layers/+emacs/org/packages.el

```diff
@@ -1025,6 +1025,12 @@ Headline^^            Visit entry^^               Filter^^                    Da
     :init
     (progn
       (add-hook 'org-mode-hook 'org-appear-mode)
+      (when (and (eq org-appear-trigger 'manual)
+                 (memq dotspacemacs-editing-style '(vim hybrid)))
+        (add-hook 'org-mode-hook
+                  (lambda ()
+                    (add-hook 'evil-insert-state-entry-hook #'org-appear-manual-start nil t)
```

Comment:
By directly modifying evil's hooks, would this also affect other buffer that's not in org-mode?

Those hooks should be made buffer-local. (via the fourth argument of add-hook)

Reply:
I must be missing something here. I thought the `t` I was passing as the final (fourth?) argument of `add-hook` did just that.